services/web3id-verifier-ts/README.md

```diff
@@ -0,0 +1,39 @@
+# Concordium dApp template
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
```

Comment:
**Request change:** The top description in the readme should probably be updated.
Documenting the purpose of this service and the endpoint exposed would be nice.

Reply:
Done, thanks for bringing this up!